syncstorage/storage/sql/queries_generic.py

```diff
@@ -163,3 +205,8 @@ def FIND_ITEMS(bso, params):
 # version using DELETE <blah> LIMIT 1000.
 PURGE_SOME_EXPIRED_ITEMS = "DELETE FROM %(bso)s "\
                            "WHERE ttl < (UNIX_TIMESTAMP() - :grace) "
+
+PURGE_TRANSACTIONS = "DELETE FROM batch_uploads WHERE batch < :batch"
+
+PURGE_TRANSACTION_CONTENTS = "DELETE FROM batch_upload_items WHERE batch < " \
+                             ":batch"
```

Comment:
Looks good, we'll also need a script to run periodically that will perform this purge.